.travis.yml

```diff
@@ -12,7 +12,7 @@ services:
   - docker
 language: go
 go:
-  - 1.12.5
+  - 1.14.x
```

Comment:
n_: can this be 1.14.1 or a specific version

Reply:
fixed to 1.14.4 , similar to maya repo